### DISCUSSION TITLE
Change termux pkg to another name

### DISCUSSION THREAD

**Discussion by Inky1003** [termux/termux-packages#7975] [Ideas]
I've installed npm's pkg and then I forgot that termux package manager is pkg also.

Can termux have another name for pkg? Or is there any way to install npm's pkg without overwriting termux pkg?

**Comment by Maxython** [comment]
This is unlikely to be done. `pkg` is the official command for installing packages.  In addition to this, `pkg` is just a bash script to manage `apt`, so you can use `apt` instead of `pkg`.

**Comment by Inky1003** [comment]
Got it. Thanks!

**Comment by asxlvm** [comment]
Hey!
You're able to change the `pkg` commands name, like this;
You type this to see the file location of your pkg script;
```
which pkg
/data/data/com.termux/files/usr/bin/pkg
```
And then you move that using the `mv` command to the same directory but you replace `pkg` with the desired name, in this example I changed the `pkg` to `packg`;
```
mv /data/data/com.termux/files/usr/bin/pkg
/data/data/com.termux/files/usr/bin/packg
```
And then as you can see it works completely fine! ;
```
packg install python
Checking availability of current mirror: ok
Hit:1 https://its-pointless.github.io/files/24.....
```

**Reply by Inky1003** [reply]
Just to fix my problem, where can I download again pkg command?

EDIT: Just found on termux-tools :P

**Reply by Efreak** [reply]
Install and use `apt-file` to find out what package includes a specific file.
